### PR TITLE
feat: test review

### PR DESCRIPTION
**Pull request by kirill-herman**
just testing

### CODE REVIEW

**Review by kirill-herman-itstep — APPROVED** [review]
gj

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # itstep-homework
 edit
```
Comment (kirill-herman-itstep) [inline README.md:2]: add more data

Comment (kirill-herman-itstep) [reply README.md:2]: done

**Review by kirill-herman-itstep — APPROVED** [review]
fjsnklasdnfk

**Review by kirill-herman-itstep — APPROVED** [review]

**Comment by AhnQiraji** [comment]
LGTM

**Review by SergeyMasalski — APPROVED** [review]

**Review by RiverBeaver — APPROVED** [review]

**Review by Poulkaw — APPROVED** [review]

**Review by DimOn-1566 — APPROVED** [review]

**Review by AhnQiraji — CHANGES_REQUESTED** [review]
YOU SHALL NOT PATH

**Review by Isuzuri — APPROVED** [review]
Alright. All works.

**Pull request closed without merging**